site/styles/_footer.scss

```diff
@@ -107,9 +110,40 @@ footer {
   .app-badge-link {
     margin: 0 5px;
   }
+
   .app-badge-image {
     height: 40px;
   }
+
+  .amazonfiretv {
```

Comment:
None of the heights on these match what is shown in Zeplin.

And none of them need to have padding or margins.